pkg/monitor/azure/nsg/nsg_test.go

```diff
@@ -475,19 +475,17 @@ func TestMonitor(t *testing.T) {
 				workerSubnet1.Properties.NetworkSecurityGroup = &nsg2
 				workerSubnet2.Properties.NetworkSecurityGroup = &nsg2
 
-				_1 := mock.EXPECT().
+				mock.EXPECT().
 					Get(ctx, resourcegroupName, vNetName, masterSubnetName, options).
 					Return(masterSubnet, nil)
 
-				_2 := mock.EXPECT().
+				mock.EXPECT().
 					Get(ctx, resourcegroupName, vNetName, workerSubnet1Name, options).
 					Return(workerSubnet1, nil)
 
-				_3 := mock.EXPECT().
+				mock.EXPECT().
 					Get(ctx, resourcegroupName, vNetName, workerSubnet2Name, options).
 					Return(workerSubnet2, nil)
-
-				gomock.InOrder(_1, _2, _3)
```

Comment:
I think I agree with dropping this. I don't know why we would care about the order. Was that your reasoning?

Reply:
Originally, things were supposed to happen in sequence (master first and then the workers).

Reply:
seems like a good change. What we care about is end behavior ("are all subnets monitored?") not programatic details ("in what order are subnets monitored?"). +1 for the change.

Reply:
Nice. That's what I thought but thanks for the context :)